static/templates/subscription_creation_form.handlebars

```diff
@@ -1,5 +1,5 @@
 <div class="hide" id="stream-creation" tabindex="-1" role="dialog"
-    aria-labelledby="stream-creation-label" aria-hidden="true">
+    aria-label="{{t 'Stream creation' }}" aria-hidden="true">
```

Comment:
does having both `aria-hidden` and `aria-label` make sense?  I guess the `aria-hidden` part is dynamic, so it does.